README.md

```diff
@@ -905,7 +940,7 @@ For example, we can*not* return `boolean|string` or `SuccessResult|FailureResult
 (if `SuccessResult` and `FailureResult` has no common class or interface; if they do, we document to return that
 interface instead).
 
-We can return `SomeClass|null` or `string|null`.
+We can return `?SomeClass` or `?string`.
```

Comment:
not sure if this change is not redundant, because in case we look at docBlock, then we use the format SomeClass|null isn't it?

Reply:
we could, but its not a docbloc, so nullable typehints used